src/components/atoms/Combo/Combo.js

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+/**
+ * combo (label / drop-down) component
+ */
+export function Combo(props) {
+  const {
+    customClass,
+    customLabelClass,
+    customSelClass,
+    id,
+    lblText,
+    required,
+    reqText,
+    options,
+    onChange,
+  } = props;
+
+  return (
+    <div
+      className={`p-4 w-full flex justify-center content-center ${customClass} ? customClass : '' `}
+    >
+      <div>
+        <label
+          className={`text-black font-semibold ${
+            customLabelClass ? customLabelClass : ""
+          } `}
+          id={id + "-label"}
+          htmlFor={id + "-select"}
+          data-cy={id + "-label"}
+          // {...rest}
+        >
+          {required ? <span className="text-red-700 font-bold">* </span> : ""}
+          {lblText}
+          {required ? (
+            <span className="text-red-800 font-bold"> ({reqText}) </span>
+          ) : (
+            ""
+          )}
+        </label>
+        <select
+          className={`w-full rounded  ${customSelClass}`}
+          id={id + "-select"}
+          data-cy={id + "-label"}
+          name={id + "-Select"}
+          onChange={onChange}
```

Comment:
you should pass in the selected option rather than the actual event. Parent components should not be aware of the implementation